packages/office-ui-fabric-react/src/components/Callout/AnnouncedCallout.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react';
+
+import { Callout } from './Callout';
+import { IAnnouncedCalloutProps } from './AnnouncedCallout.types';
+import { DelayedRender } from '../../Utilities';
+
+/**
+ * A special Callout that announces its contents regardless of focus
+ * @param props - Props for the component
+ */
+export const AnnouncedCallout: React.StatelessComponent<IAnnouncedCalloutProps> = (props: IAnnouncedCalloutProps): JSX.Element => {
+  // announced scenario requires specific role and ariaLive values regardless of user provided values
+  const announcedProps = {
+    role: 'status',
+    ariaLive: 'assertive'
+  };
+  return (
+    <Callout {...announcedProps} {...props}>
+      <DelayedRender>
```

Comment:
Can we not just use an Announced inside a Callout? (Or add an example of doing that to the Callout docs?)

Reply:
Yes, this is 100% doable without this control. I have a demo right here:
https://codepen.io/micahgodbolt/pen/abbxvPN?editors=0010

The goal is to provide a consistent solution for a very common scenario. It's very similar to the FocusTrapCallout. 

The advantage of this approach is that we provide solutions for common problems without increasing the weight of the original callout (would be silly to ship FocusTrapZone and DelayedRender with every callout).

I agree, there is technical debt in creating this control vs just writing an example, but seeing as it solves a major accessibility issues, I feel that it is a worthy addition.